### DISCUSSION TITLE
The requirements of Dictionary transforms in Compose

### DISCUSSION THREAD

**Discussion by Jingnan-Jia** [Project-MONAI/MONAI#2770] [General]
Hi, from [here](https://github.com/Project-MONAI/MONAI/blob/7ea09bf0aa7604a7dcb96794699915bbcf6cdab9/monai/transforms/compose.py#:~:text=pass-through,semantics) I noted that the customed dictionary transforms **must have pass-through semantics**, can you give more explanication on it? What is 'pass-through sementics' ?

**Comment by Nic-Ma** [comment]
Hi @Jingnan-Jia ,

That mean: unused values in the dictionary must be copied to the return dictionary.
I will change the ";" to ":" later in some other doc-string related PR.

Thanks.

**Reply by Jingnan-Jia** [reply]
Thanks, it's clearer.
